src/components/manage/Controlpanels/Aliases.jsx

```diff
@@ -61,635 +60,541 @@ const filterChoices = [
-                        lastItem={null}
-                        siblingRange={1}
-                        totalPages={this.state.pages}
-                        onPageChange={(e, o) => this.handlePageChange(e, o)}
-                      />
-                    )}
-                    <Menu.Menu
-                      position="right"
-                      style={{ display: 'flex', marginLeft: 'auto' }}
-                    >
-                      <Menu.Item style={{ color: 'grey' }}>
-                        <FormattedMessage id="Show" defaultMessage="Show" />:
-                      </Menu.Item>
-                      {map(itemsPerPageChoices, (size) => (
-                        <Menu.Item
-                          style={{
-                            padding: '0 0.4em',
-                            margin: '0em 0.357em',
-                            cursor: 'pointer',
-                          }}
-                          key={size}
-                          value={size}
-                          active={size === this.state.itemsPerPage}
-                          onClick={(e, o) => this.handleItemsPerPage(e, o)}
-                        >
-                          {size}
-                        </Menu.Item>
+                  />
+                </Form.Field>
+                <Button onClick={() => handleSubmitFilter()} primary>
+                  Filter
+                </Button>
+                <Header size="small">
+                  <FormattedMessage
+                    id="Alternative url path → target url path (date and time of creation, manually created yes/no)"
+                    defaultMessage="Alternative url path → target url path (date and time of creation, manually created yes/no)"
+                  />
+                </Header>
+
+                <Table>
+                  <Table.Body>
+                    <Table.Row>
+                      <Table.HeaderCell>
+                        <FormattedMessage id="Select" defaultMessage="Select" />
+                      </Table.HeaderCell>
+                      <Table.HeaderCell>
+                        <FormattedMessage id="Alias" defaultMessage="Alias" />
+                      </Table.HeaderCell>
+                      <Table.HeaderCell>
+                        <FormattedMessage id="Target" defaultMessage="Target" />
+                      </Table.HeaderCell>
+                      <Table.HeaderCell>
+                        <FormattedMessage id="Date" defaultMessage="Date" />
+                      </Table.HeaderCell>
+                      <Table.HeaderCell>
+                        <FormattedMessage id="Manual" defaultMessage="Manual" />
+                      </Table.HeaderCell>
+                    </Table.Row>
+                    {aliases.items.length > 0 &&
+                      aliases.items.map((alias, i) => (
+                        <Table.Row key={i}>
+                          <Table.Cell>
+                            <Checkbox
```

Comment:
For some reason, the checkboxes no longer works when selecting multiple alias urls.